doc/conf_ref.rst

```diff
@@ -258,9 +258,9 @@ or :ref:`mirrorlist <mirrorlist-label>` option definition.
 ``log_rotate``
     :ref:`integer <integer-label>`
 
-    Log files are rotated log_rotate times before being removed. If log_rotate
-    is 0, the rotation is not performed.
-    Default is 4.
+    Log files are rotated ``log_rotate`` times before being removed. If log_rotate
```

Comment:
One more quotation of `log_rotate` at the end of the line.

Reply:
Fixed